src/gafaelfawr/factory.py

```diff
@@ -420,43 +420,6 @@ def create_provider(self) -> Provider:
             # This should be caught during configuration file parsing.
             raise NotImplementedError("No authentication provider configured")
 
-    def create_user_info_service(self) -> UserInfoService:
```

Comment:
Apologies, this is just a code move. I forgot I did that or I would have merged it separately.